src/Analyzers/CSharp/Tests/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -1494,6 +1496,22 @@ public async Task PropertyIsIncrementedAndValueDropped()
             await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
+        [WorkItem(43191, "https://github.com/dotnet/roslyn/issues/43191")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
+        public async Task PropertyIsIncrementedAndValueDropped_VerifyAnalizerMessage()
+        {
+            var code = @"class MyClass
+{
+    private int P { get; set; }
+    public void M1() { ++P; }
+}";
+
+            await VerifyCS.VerifyAnalyzerAsync(code, new DiagnosticResult(
+                CSharpRemoveUnusedMembersDiagnosticAnalyzer.s_removeUnreadMembersRule)
+                .WithSpan(3, 17, 3, 18)
+                .WithArguments("MyClass.P"));
+        }
```

Comment:
so can we have the variants of these tests where the read does happen somewhere, and then we don't get these diagnostics?

Reply:
I don't understand. What kind of test should I add?

Reply:
One that has the same code as above, but also reads the property somewhere else, so we don't get the diagnostic.

Reply:
@DoctorKrolic I think what @CyrusNajmabadi meant was to add another method that reads the property without incrementing it, or do this somewhere else in the same method. Not to have the property in another class, although that could be a useful test as well.

Reply:
I would also add a test where the result of the increment operation is actually used, for example:
```c#
    public int M1() { return ++P; }
```

Reply:
still looking for a test with the same contents of above, but with an additional usage of the property showing that we then won't emit this diagnostic in that case.

Reply:
@Neme12 return of an increment was already tested